packages/lib-user/package.json

```diff
@@ -0,0 +1,57 @@
+{
+  "name": "@zooniverse/user",
+  "description": "A library for Zooniverse user and user group pages",
+  "license": "Apache-2.0",
+  "author": "Zooniverse <contact@zooniverse.org> (https://www.zooniverse.org/)",
+  "version": "0.0.0",
+  "main": "dist/cjs/index.js",
+  "exports": {
+    ".": {
+      "import": "./dist/esm/index.js",
+      "require": "./dist/cjs/index.js"
+    },
+    "./*": {
+      "import": "./dist/esm/*/index.js",
+      "require": "./dist/cjs/*/index.js"
+    }
+  },
+  "sideEffects": false,
```

Comment:
related to this library in ESM format - add `"type": "module"` and refactor package.json